### ISSUE TITLE
[Bug] "crypto -> top" error: This API key is invalid

### ISSUE THREAD

**Issue by arisliang** [DidierRLopes/GamestonkTerminal#416]
**Describe the bug**

Unable to run crypto -> top command, an error is given: 
>"This API Key is invalid.": {}

**To Reproduce**
1. crypto
2. top

**Screenshots**
![image](https://user-images.githubusercontent.com/1115503/116723375-0a11bd00-aa12-11eb-84af-a3e321c604e9.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04
 - Python version: 3.6.13

**Additional context**
Do I need to specify the API key for coinmarketcap? I have the key, but README (https://github.com/DidierRLopes/GamestonkTerminal#api-keys) doesn't list it as necessary.

**Comment by jmaslek** [comment]
Yes its necessary (hence why you get that the API key is invalid).  I will update the list
